gocelery_test.go

```diff
@@ -192,6 +224,56 @@ func TestRegister(t *testing.T) {
 	}
 }
 
+// RUN this test with -race flag to see if there are race conditions
+func TestCeleryWorker_RunTaskTaskCopy(t *testing.T) {
```

Comment:
This will fail with `-race` flag if a task copy is not made

```
=== RUN   TestCeleryWorker_RunTaskTaskCopy
2018/08/26 18:58:09 bff7274f-991a-4cfb-8b98-1afc16c748ec
==================
WARNING: DATA RACE
Write at 0x00c420092908 by goroutine 8:
  github.com/gocelery/gocelery.(*stateFulTask).ParseKwargs()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/gocelery_test.go:71 +0x41
  github.com/gocelery/gocelery.(*CeleryWorker).RunTask()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/worker.go:126 +0x119
  github.com/gocelery/gocelery.(*CeleryWorker).StartWorker.func1()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/worker.go:56 +0x106

Previous write at 0x00c420092908 by goroutine 7:
  github.com/gocelery/gocelery.(*stateFulTask).ParseKwargs()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/gocelery_test.go:71 +0x41
  github.com/gocelery/gocelery.(*CeleryWorker).RunTask()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/worker.go:126 +0x119
  github.com/gocelery/gocelery.(*CeleryWorker).StartWorker.func1()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/worker.go:56 +0x106

Goroutine 8 (running) created at:
  github.com/gocelery/gocelery.(*CeleryWorker).StartWorker()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/worker.go:39 +0xd9
  github.com/gocelery/gocelery.(*CeleryClient).StartWorker()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/gocelery.go:43 +0x4e
  github.com/gocelery/gocelery.TestCeleryWorker_RunTaskTaskCopy()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/gocelery_test.go:234 +0x43f
  testing.tRunner()
      /Users/vimukthi/software/go/go1.10.3/src/testing/testing.go:777 +0x16d

Goroutine 7 (running) created at:
  github.com/gocelery/gocelery.(*CeleryWorker).StartWorker()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/worker.go:39 +0xd9
  github.com/gocelery/gocelery.(*CeleryClient).StartWorker()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/gocelery.go:43 +0x4e
  github.com/gocelery/gocelery.TestCeleryWorker_RunTaskTaskCopy()
      /Users/vimukthi/projects/go/src/github.com/gocelery/gocelery/gocelery_test.go:234 +0x43f
  testing.tRunner()
      /Users/vimukthi/software/go/go1.10.3/src/testing/testing.go:777 +0x16d
==================
2018/08/26 18:58:09 fcc7b5f4-f4cd-4d4a-bf4e-4c837a105683
--- FAIL: TestCeleryWorker_RunTaskTaskCopy (0.11s)
	testing.go:730: race detected during execution of test
FAIL

```